@mtreinish GHA linter workflow reported the following error. 

```
Run pylint ./python
************* Module /__w/qrmi/qrmi/.pylintrc
.pylintrc:1:0: W0012: Unknown option value for '--disable', expected a valid pylint message and got 'too-many-positional-arguments' (unknown-option-value)
************* Module qrmi
python/qrmi/__init__.py:17:0: E0401: Unable to import 'qrmi._core' (import-error)

```
Do you think ignoring import error is reasonable to change ? 

`from ._core import *  # pylint: disable=import-error`

--- a/python/qrmi/__init__.py
+++ b/python/qrmi/__init__.py
@@ -14,4 +14,4 @@
 
 """QRMI python package"""
 
-from ._core import *
+from ._core import *  # pylint: disable=import-error

